src/Actions.js

```diff
@@ -26,24 +34,23 @@ function joinGroupMe(gmToken, callback){
       
       .then((resp) => {
         if (resp.ok){
-          callback()
+          //can't test this due to how we have the GM authentication set up
+          addEvent(userId, googleAccessToken, gmToken);
+          callback();
```

Comment:
adding an event should be purely backend. When the 'addtogroupme' api is called, you can add an event to the DB

Reply:
(in otherwords, move this functionality to gmail.py)